build.gradle

```diff
@@ -17,6 +17,8 @@ repositories {
 dependencies {
     implementation 'com.itextpdf:itextpdf:5.5.10'
     implementation 'org.jpedal:jpedal-lgpl:4.74b27'
+    implementation 'org.bouncycastle:bcprov-jdk15on:1.49'
+    implementation 'org.bouncycastle:bcpkix-jdk15on:1.49'
```

Comment:
I tried without this second dependency and it seems like loading a PW protected pdf also works without this dependency. Do you know what it is used for or can we remove it?